lib/brains/views/paymethod.py

```diff
@@ -14,55 +14,73 @@
 log = getLogger('s.brains')
 
 
-@api_view(['POST'])
-def create(request):
-    client = get_client().PaymentMethod
-    form = PaymentMethodForm(request.DATA)
+class Braintree(APIView):
 
-    if not form.is_valid():
-        raise FormError(form.errors)
+    def patch(self, request):
+        client = get_client().PaymentMethod
+        form = PayMethodUpdateForm(request.DATA)
 
-    buyer = form.buyer
-    braintree_buyer = form.braintree_buyer
-    result = client.create(form.braintree_data)
-
-    if not result.is_success:
-        log.warning('Error on creating Payment method: {0}, {1}'
-                    .format(buyer.uuid, result.message))
-        raise BraintreeResultError(result)
+        if not form.is_valid():
+            raise FormError(form.errors)
 
-    braintree_method = result.payment_method
-    log.info('PaymentMethod created for: {0}'.format(buyer.uuid))
+        solitude_method = form.cleaned_data['paymethod']
 
-    solitude_method = BraintreePaymentMethod.objects.create(
-        braintree_buyer=braintree_buyer,
-        type=PAYMENT_METHOD_CARD,
-        type_name=braintree_method.card_type,
-        provider_id=braintree_method.token,
-        truncated_id=result.payment_method.last_4
-    )
-    log.info('Method {0} created.'.format(solitude_method.pk))
+        client = get_client().PaymentMethod
+        result = client.delete(solitude_method.provider_id)
```

Comment:
shouldn't this only delete from Braintree when the patch is setting it to inactive? Or maybe this should be in a ``delete`` handler instead of patch?